src/scripts/main.js

```diff
@@ -1,3 +1,141 @@
 'use strict';
 
-// write code here
+const table = document.querySelector('table');
+const tableBody = document.createElement('tbody');
+table.innerHTML = '';
+table.append(tableBody);
+
+const tbody = document.querySelector('tbody');
+
+const apppendRow = document.querySelector('.append-row');
+const removeRow = document.querySelector('.remove-row');
+const appendColumn = document.querySelector('.append-column');
+const removeColumn = document.querySelector('.remove-column');
+
+const tableSize = {
+  width: 4,
+  height: 4,
+};
+
+const buttonsState = {
+  apppendRowButton: 'enabled',
+  removeRowButton: 'enabled',
+  appendColumnButton: 'enabled',
+  removeColumnButton: 'enabled',
+};
```

Comment:
I'm not sure that this state is mandatory

Reply:
since it is the same info that you can get by read disabled property of each button